common/bytes.go

```diff
@@ -132,3 +135,10 @@ func LeftPadBytes(slice []byte, l int) []byte {
 
 	return padded
 }
+
+// intToByte converts value of int to []byte
+func IntToByte(num uint64) []byte {
```

Comment:
We need to make sure the byte order. 
@winnie-byun Could you rename the function with "littleEndian" or refine the function to get the byte order type as a parameter?

Reply:
I renamed the function.